projects/aca-content/src/lib/components/knowledge-retrieval/search-ai/agents-button/agents-button.component.scss

```diff
@@ -0,0 +1,48 @@
+aca-agents-button.aca-agents-button {
+  height: 32px;
+  display: block;
+
+  button {
+    &.aca-agents-button-animated {
+      &.aca-agents-button-menu-trigger {
+        height: auto;
+        cursor: pointer;
+        border: none;
+        background: transparent;
+        width: max-content;
+        padding: 0 4px 0 0;
+
+        ng-lottie {
+          line-height: normal;
+        }
+      }
+    }
+  }
+}
+
+.aca-agents-button-menu {
+  div:not(:empty) {
+    padding-top: 8px;
+    padding-bottom: 6px;
+  }
+
+  .aca-agents-button-menu-list {
+    margin-left: -6px;
+    padding-top: 0;
+
+    &-agent {
+      height: 40px;
+
+      &-avatar {
+        vertical-align: middle;
+        margin-bottom: 2px;
+        margin-right: 12px;
+        border-radius: 50%;
+        background-color: #9acd9a; // temporary to keep place for avatar, will be replaced with avatar image during synchronization with backend
```

Comment:
we now have the `adf-avatar` component that shows image or fallback to initials, maybe it can be used in this feature

Reply:
replaced with adf-avatar now